### PR TITLE
 adds background task param to StreamingResponse and FileResponse

### PR DESCRIPTION
**Pull request by zya-performance**
https://github.com/encode/starlette/issues/166

### CODE REVIEW

**Comment by lovelydinosaur** [comment]
Good call here, thanks. Missing test coverage at the moment, plus @alexbotello's comment is correct.

**Comment by alexbotello** [comment]
@zya-performance make sure to run Black against changed files to pass integration tests

**Comment by lovelydinosaur** [comment]
Yup, as @alexbotello mentioned, running `scripts/lint` will clean up those bits.

**Comment by lovelydinosaur** [comment]
Reminder to self: we ought to have a “contributing” docs section! :)

**Comment by zya-performance** [comment]
> Yup, as @alexbotello mentioned, running `scripts/lint` will clean up those bits.

did that, and it changed all the files, I committed only the files I changed, But it still fails on Travis.!!  Any way to make that same check locally before It fails on the CI ?

**Comment by zya-performance** [comment]
> Reminder to self: we ought to have a “contributing” docs section! :)

:) yes! This will be very useful.

File: tests/test_responses.py [hunk 1]
```diff
@@ -2,10 +2,9 @@
 import os
 
 import pytest
-
```
[the remark this answers is not in the document]
Comment (zya-performance) [reply tests/test_responses.py]: Yippiee Yay! finally  done with this.

**Comment by lovelydinosaur** [comment]
👍

**Pull request merged**